### DISCUSSION TITLE
TypeError: Cannot read property 'getPaths' of null

### DISCUSSION THREAD

**Discussion by Basit-qc** [atom/atom#23463] [Q&A]
Dcoumentation says this 
![image](https://user-images.githubusercontent.com/9863016/148463625-c85ddc5e-459e-4f5e-a388-70f43e9604ad.png)

Works fine for most of the user but getting ```TypeError: Cannot read property 'getPaths' of null```
on this line ```atom.project.getPaths()[0]```

from a Windows user. 

When can `atom.project` be null?

**Comment by icecream17** [comment]
The only time `atom.project === null` is if `atom.destroy()` was called:
https://github.com/atom/atom/blob/7033a5a22f260a54c0ffb69f45d12622478778df/src/atom-environment.js#L446-L461

**Reply by Basit-qc** [reply]
Thank you @icecream17
